src/System.Numerics.Vectors/tests/GenericVectorTests.cs

```diff
@@ -2699,11 +2699,12 @@ private static bool AreSameInfinity(double d1, double d2)
             }
         }
 
-        internal static T[] GenerateRandomValuesForVector<T>() where T : struct
+        internal static T[] GenerateRandomValuesForVector<T>(int numValues = int.MinValue) where T : struct
```

Comment:
These changes need to be in the `.tt` file as well, right?

Reply:
Done. Yes I missed it!!!